ecosystem/contributor-awards-2023.html

```diff
@@ -0,0 +1,145 @@
+          </div>
+          <div class="col-md-6">
+            <h2>PyTorch 2023 Awardees</h2>
+          </div>
+        </div>
+      </div>
+
+      <table class="table mb-5 table-bordered">
+        <tr>
+          <td>PY-TORCHBEARER</td>
+          <td>Excellence in Long-Term Contributions Across All Modalities</td>
+          <td>Jeffy Daily and Jack Cao</td>
+        </tr>
+        <tr>
+          <td>PYTORCH PACE-SETTER</td>
+          <td>Excellence in High-Level Activity and Contributions</td>
+          <td>James Keeble</td>
+        </tr>
+        <tr>
+          <td>PYTORCH NEWCOMER</td>
+          <td>Excellence in New Contributions</td>
+          <td>Tim Dettmers and Srishti Gureja</td>
+        </tr>
+        <tr>
+          <td>PYTORCH AMBASSADOR</td>
+          <td>Excellence in Bringing New Users to the Community</td>
+          <td>Mazen Alotaibi (Ma7dev), Stas Bekman, and Junghwan Park</td>
+        </tr>
+        <tr>
+          <td>PYTORCH SUPERHERO</td>
+          <td>Excellence in All Aspects of Community Contributions</td>
+          <td>Piotr Bialecki (ptrblck)</td>
+        </tr>
+        <tr>
+          <td>PYTORCH REVIEW POWERHOUSE</td>
+          <td>Excellence in Code Review</td>
+          <td>Aaron Gokaslan</td>
+        </tr>
+        <tr>
+          <td>PYTORCH PROBLEM-SOLVER</td>
+          <td>Excellence in Uncovering or Resolving Bugs</td>
+          <td>Vadim Kantorov</td>
+        </tr>
+        <tr>
+          <td>PYTORCH INNOVATOR</td>
+          <td>Excellence in Innovative New Features or Approaches</td>
+          <td>Emcastillo</td>
+        </tr>
+        <tr>
+          <td>PYTORCH TRAIL-BLAZER</td>
+          <td>Excellence in Documentation and Knowledge Sharing</td>
+          <td>José Luis Castro Garcia</td>
+        </tr>
+      </table>
+      
+      <div class="container mt-4 mb-3">
+        <div class="row align-items-center justify-content-md-center">
+          <div class="col-md-2">
+            <img style="max-width: 150px;" src="/assets/images/contributor-awards-2023/pt-nominee.png" alt="Nominee badge">
+          </div>
+          <div class="col-md-6">
+            <h2>PyTorch 2023 Nominees</h2>
+          </div>
+        </div>
+      </div>
+
+      <table class="table mb-5 table-bordered">
+        <tr>
+          <td>Bowen Bao</td>
+          <td>Li-Huai (Allan) Lin</td>
+          <td>Thomas Viehmann</td>
+
+        </tr>
+        <tr>
+          <td>Yuanyuan Chen</td>
+          <td>Li Ning</td>
+          <td>Antoni Viros-i-Martin</td>
+
+        </tr>
+        <tr>
+          <td>Justin Chu</td>
+          <td>Muhammad Qasim Khan</td>
+          <td>Christian von der Weth</td>
+
+        </tr>
+        <tr>
+          <td>K. Frank</td>
+          <td>Fritz Obermeyer</td>
+          <td>Ivan Yashchuk</td>
+
+        </tr>
+        <tr>
+          <td>Jeremian Johnson</td>
+          <td>Kulin Seth</td>
+          <td>Xiobing Zhang</td>
```

Comment:
@cjyabraham , I think it should be Xiaobing Zhang, could you help double check it? Thanks!

Reply:
Sure. This is fixed: https://pytorch.org/ecosystem/contributor-awards-2023